src/runtime/tests/manifest-test.ts

```diff
@@ -376,15 +376,15 @@ ${particleStr1}
         x: writes S
 
       recipe
-        handle1: use
-        handle2: use
+        handle1: use *
+        handle2: use *
         P1
           x: writes handle1
         P1
           x: writes handle2
       recipe
-        handle1: use
-        handle2: use
+        handle1: use *
+        handle2: use *
```

Comment:
shall we add more tests for this (or maybe in manifest-parser-test.ts?) - for having different variants of recipe handles (with/without names/ids/* etc) successfully parsed?

Reply:
Thanks Maria, that'd be really good to do. I'd like to prioritize stability, error messages and general robustness but I don't think I'll be able to until after slandles due to the number of tests and particles to be converted and the need for slandles v1 support.